src/OverlayTrigger.js

```diff
@@ -281,19 +281,21 @@ const OverlayTrigger = React.createClass({
 
   _getContainerDimensions() {
     const containerNode = this.getContainerDOMNode();
-    let width, height;
+    let width, height, scroll;
+
     if (containerNode.tagName === 'BODY') {
       width = window.innerWidth;
       height = window.innerHeight;
+      scroll =
+        domUtils.ownerDocument(containerNode).documentElement.scrollTop ||
```

Comment:
Understandable. In that case it all LGTM :+1: 

--
What do you think about this cosmetic stroke:
```js
if (containerNode.tagName === 'BODY') {
  width = window.innerWidth;
  height = window.innerHeight;
  scroll = domUtils.ownerDocument(containerNode).documentElement.scrollTop
} else {
  width = containerNode.offsetWidth;
  height = containerNode.offsetHeight;
}

scroll = scroll || containerNode.scrollTop;
```
or even
```js
return {width, height, scroll || containerNode.scrollTop};
```
I'm totally OK with your code. :cherries: 
It's because I'm learning and trying different coding patterns to get any feedback.